e2e/basic/bittwister_test.go

```diff
@@ -206,31 +182,11 @@ func TestBittwister_Packetloss(t *testing.T) {
 			targetPacketlossRate: 10,
 			tolerancePercent:     50,
 		},
-		{
```

Comment:
One these ones I would maybe suggest 3 tests. 
1. 0% loss
2. random loss
3. 100% loss

you could then set the random loss to be 1-99.

Reply:
0 and 100 percent loss to cover edge cases, unless those are not accepted. 
in which case I would do 1% and 99% with the random loss or something.

Reply:
We already had 100% and when we set it, it blocks all packets and the pod is not accessible; therefore, we cannot reach to the pod and measure anything and most of the times k8s API fails to perform anything on that pod.

Setting to `0%` in fact disables the packetloss service for that pod, so not sure if we test anything there.

If we wann go for a random value we should do something between 10% and 90% as the measurement accuracy falls down when we reduce it to near zero or increase it to near 100%.